.github/workflows/assignments_test_docker.yml

```diff
@@ -21,12 +26,14 @@ jobs:
       - name: Build Docker image
         working-directory: ./weblab-docker
         env:
-          BUILD_ARGS: "--build-arg ANDY_BRANCH=${{ github.sha }}"
+          BUILD_ARGS: "--build-arg ANDY_BRANCH=${{ env.GHA_SHA }} --build-arg ANDY_REPO=${{ env.GHA_REPO }}"
         run: |
           make
 
       - name: Run the reference solutions of all assignments and verify the scores are 100/100
         env:
-          COMMIT_HASH: ${{ github.sha }}
+          # github.repository contains owner, we don't want that
+          REPO_NAME: ${{ github.event.repository.name }}
```

Comment:
Instead of passing the repository name to the script in order to find the correct folder, could we use the "path" option of the checkout action to force it to always clone the repo to a directory called "andy" ? https://github.com/actions/checkout#usage

Reply:
I understood `path` to be the working directory in which `git clone` is issued because the documentation is abhorrent. It is possible to change it, but I feel this way is more robust -- we should not be relying on hardcoding `andy` anywhere really, in my opinion. Let me know what to do.

Reply:
I see, let's keep it as is then. Thanks!